build.sbt

```diff
@@ -74,6 +75,18 @@ lazy val scalaApi = (project in file("scala-api"))
   )
   .dependsOn(api, jmx % "test")
 
+lazy val scalaEffectApi = (project in file("scala-effect-api"))
+  .settings(
+    commonSettings,
+    scalaSettings,
+    scalacOptions += "-language:higherKinds",
+    name := "metrics-scala-effect",
```

Comment:
i would call it `...-cats-effect2`, or `-ce2` as we will probably need a `ce3` or `-cats-effect3` in the future